backend/core/src/listings/views/config.ts

```diff
@@ -41,6 +41,22 @@ const views: Views = {
       "units.status",
       "unitType.id",
       "unitType.name",
+      "unitsSummary.id",
+      "summaryUnitType.name",
+      "unitsSummary.monthlyRent",
+      "unitsSummary.monthlyRentAsPercentOfIncome",
+      "unitsSummary.amiPercentage",
+      "unitsSummary.monthlyIncomeMin",
+      "unitsSummary.monthlyIncomeMax",
+      "unitsSummary.maxOccupancy",
+      "unitsSummary.minOccupancy",
+      "unitsSummary.floorMin",
+      "unitsSummary.floorMax",
+      "unitsSummary.sqFeetMin",
+      "unitsSummary.sqFeetMax",
+      "priorityType.id",
+      "unitsSummary.totalCount",
+      "unitsSummary.totalAvailable",
```

Comment:
I'm just talking out loud, to make sure I understand correctly: the reason we put this in the "base" view is because both the "full listings" and the "individual listing" pages use the "base" view:

![image](https://user-images.githubusercontent.com/8754454/130512548-913f0907-a7ae-4e59-aa6a-59e6141a08ce.png)

Does that sound right to you?

Reply:
This is very recently added, but we now have 3 different "views" of the listing data: `base`, `detail`, and `full` (in increasing order of amount of data).

`Detail` inherits all the selects from `base`, which is why I didn't have to add anything to the `detail` view. A quirk of this is that `full` doesn't inherit from `base`, I think just because it was much easier to add `leftJoinAndSelect` than to manually add all the `selects` for every single field. So that's why I had to update the `full` view as well.

I'm fairly certain that everything is currently using the `full` view, so it might be overkill to add this to `base`, but it made sense to me that summary data should be included in the `base` view, even if it isn't being used yet.

Reply:
Thanks for writing this up! It does actually look like we are using the `base` view in at least the individual-listing page: https://github.com/CityOfDetroit/bloom/blob/33c9bff42b8ec4ac407ae8231472fd4262a0c410/sites/public/pages/listing/%5Bid%5D/%5Bslug%5D.tsx#L46. So SGTM to add these to the `base` view!